src/util/namespaceUtils.ts

```diff
@@ -64,8 +65,12 @@ export class NamespaceUtils {
   }
 
   static calculateDuration = (durationInDay: number): number =>{
+    let chainConfig = new ChainProfileConfig(networkState.chainNetworkName);
+    chainConfig.init();
+    let blockTargetTime = parseInt(chainConfig.blockGenerationTargetTime);
+    let blockTargetTimeByDay = (60 / blockTargetTime) * 60 * 24;
```

Comment:
this calculation is wrong actually, should get number of seconds per day / blockTargetTime, then floor the result to integer

Reply:
They are the same? 60s / 15s x 60 min x 24 hrs for each day

Reply:
what if the time set more than 60s ? or what if it is set to 13s ? you did not handle it, the final result will be in float